moveit_kinematics/cached_ik_kinematics_plugin/CMakeLists.txt

```diff
@@ -20,6 +19,10 @@ generate_parameter_library(
   include/moveit/cached_ik_kinematics_plugin/cached_ik_kinematics_parameters.yaml # path to input yaml file
 )
 
+target_link_libraries(moveit_cached_ik_kinematics_base
```

Comment:
While you're touching this code, please specify `PRIVATE` or `PUBLIC` for these link libraries. MoveIt needs to transition to using those keywords on all `target_link_libraries` calls.

Reply:
Using keywords in 68a6ee278